python/ray/data/dataset.py

```diff
@@ -4218,6 +4218,12 @@ def _aggregate_result(self, result: Union[Tuple, Mapping]) -> U:
         else:
             return result
 
+    @PublicAPI(stability="alpha")
```

Comment:
```suggestion
```

This isn't necessary.

Reply:
removed

Reply:
For my own understanding, why don't we need this? We use `@PublicAPI(stability="alpha")` for other `Dataset` methods, and I'm not sure how else we'd distinguish that this particular method is unstable.

Reply:
I think this is simple enough we can immediately graduate this to Public.